client/source/js/modules/analysis/optimization-ctrl.js

```diff
@@ -275,10 +277,129 @@ define(['./module', 'angular', 'd3'], function (module, angular, d3) {
       }
     };
 
+    function validateYears(){
+      return $scope.params.objectives!==undefined &&
+             $scope.params.objectives.year!==undefined &&
+             $scope.params.objectives.year.start!==undefined &&
+             $scope.params.objectives.year.end !==undefined &&
+             ($scope.params.objectives.year.start < $scope.params.objectives.year.end);
```

Comment:
It allows start '' (empty string) and end with any value.
To reproduce:
1. Set start/end years,
2. Remove `startYear` value
3. Switch to tab 3
4. See the error